packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -28,16 +29,33 @@ export const compareVariableByName = (a: DesignTokenVariableStructure, b: Design
 
 /**
  * Sort Token variable by name using the local alphabetical order
- * @param _variableSet Complete set of the parsed Design Token
+ * @param _variables Complete set of the parsed Design Token
+ * @param options Parameters of the Design Token list transform function
  */
-export const getTokenSorterByName: TokenListTransform = (_variableSet) => (tokens) => tokens.sort((a, b) => a.getKey().localeCompare(b.getKey()));
+export const getTokenSorterByName: DesignTokenListTransform = (_variables, options) => {
```

Comment:
I am a bit confused by the unused parameter _variables. Couldn't it be removed ?

Reply:
Can not be removed (because of the `options` next parameter that is used).
The only solution is to rename it `_` but it will be ugly in the jsdoc :S